addons/misra.py

```diff
@@ -46,19 +59,20 @@ def reportError(location, num1, num2):
         elif len(ruleTexts) == 0:
             errmsg = 'misra violation (use --rule-texts=<file> to get proper output) [' + id + ']'
         else:
-            return
-        errmsg = '[' + location.file + ':' + str(location.linenr) + '] (style): ' + errmsg + '\n'
-        sys.stderr.write(errmsg)
+            errmsg = 'misra violation (no description found for rule) [' + id + ']'
```

Comment:
I want that you fix this or restore it. how do I suppress the error now by removing the text from the file.

I want that the rule is hidden by default if the text is missing. but if some command line flag is used such message can be written. for instance `--check-rule-texts`..

To me it would seem better that when `--check-rule-texts` is used it will report all missing rule texts (for implemented checkers). So I envision that this will not be used in reportError() but there will be a separate function for this that works a little like `generateTable()` but it will match existing checks against the available rule texts.

Reply:
actually.. I recommend that `--check-rule-texts` is added in a separate PR. so please just restore the return.